pkg/sdk/tags_integration_test.go

```diff
@@ -0,0 +1,81 @@
+package sdk
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestInt_TagCreate(t *testing.T) {
+	client := testClient(t)
+
+	databaseTest, databaseCleanup := createDatabase(t, client)
+	t.Cleanup(databaseCleanup)
+	_, schemaCleanup := createSchema(t, client, databaseTest)
+	t.Cleanup(schemaCleanup)
+
+	ctx := context.Background()
+	t.Run("create with comment", func(t *testing.T) {
+		name := randomAccountObjectIdentifier(t)
+		comment := randomComment(t)
+		err := client.Tags.Create(ctx, NewCreateTagRequest(name).WithOrReplace(true).WithComment(&comment))
```

Comment:
In multiple tests, you create a tag and add a cleanup function for it. This could be extracted as a helper function for these tests. I did something similar in `database_role_integration_test.go` (`createDatabaseRole` func).

It would be nice to see a similar thing here.

Reply:
done